app/controllers/api/topics_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Api  
+  class TopicsController < ActionController::Base
+    include DeviseTokenAuth::Concerns::SetUserByToken
+    before_action :authenticate_api_user!
+
+    def index
+      @topics = Topic.all
```

Comment:
What about adding a test for this?

Reply:
I'll merge the other tests before that in order to use all the RSpec directories